.github/workflows/autopr.yml

```diff
@@ -1,23 +1,32 @@
 on:
   push:
     branches:
-    - feature-1
+    - dev
```

Comment:
WHY did you change the branch you're branch name?

Reply:
The branch name have changed in the repo